rest/config_test.go

```diff
@@ -799,3 +800,56 @@ func TestParseCommandLineWithConfigContent(t *testing.T) {
 	assert.False(t, guest.Disabled)
 	assert.Equal(t, base.SetFromArray([]string{"*"}), guest.ExplicitChannels)
 }
+
+// This test uses both test_data_bucket and test_indexbucket buckets; these buckets should be available
+// on Couchbase server instance before running this test.
+func TestValidateAndAddDBFromConfig(t *testing.T) {
+	if base.UnitTestUrlIsWalrus() {
+		t.Skip("ValidateAndAddDBFromConfig test require Couchbase Bucket")
+	}
+	tmpDir, err := ioutil.TempDir("", "sync_gateway")
+	require.NoError(t, err, "Error whilst creating temporary directory")
+	configFile, err := ioutil.TempFile(tmpDir, "duplicate.bucket.conf")
+	require.NoError(t, err, "Error whilst creating temporary config file")
+
+	content := `{"logging":{"console":{"log_keys":["*"]}},"databases":{"db1":{"server":"couchbase://localhost",
```

Comment:
localhost for a running Couchbase Server is not a valid assumption, given it can be customised by SG_TEST_COUCHBASE_SERVER_URL.

You can use `base.UnitTestUrl()` to get the URL.

Reply:
Fixed in latest revision.